tests/CalendarTest.php

```diff
@@ -107,39 +107,6 @@ public function setUp(): void
         );
     }
 
-    /**
-     * @test
-     */
-    public function time_stamps_need_to_have_type_time_stamp(): void
```

Comment:
I would keep the tests, what if PHPStan would be removed from the project?

Reply:
restored tests